src/pages/tutorial/task/task.js

```diff
@@ -234,8 +205,20 @@ class TaskPage extends React.Component {
     const currentThreadProgress = this.getStoredProgressForCurrentTask();
     const isNoChecked = currentThreadProgress[blockId] !== undefined && !currentThreadProgress[blockId];
     const isYesChecked = currentThreadProgress[blockId] !== undefined && !!currentThreadProgress[blockId];
+
+    let verificationClasses = 'alert integr8ly-alert integr8ly-module-column--steps_alert-blue';
+    let verificationIcon = 'integr8ly-alert-icon far fa-circle';
+    if (isYesChecked) {
+      verificationClasses = 'alert integr8ly-alert integr8ly-module-column--steps_alert-green';
+      verificationIcon = 'integr8ly-alert-icon far fa-check-circle';
+    }
+    if (isNoChecked) {
+      verificationClasses = 'alert integr8ly-alert integr8ly-module-column--steps_alert-red';
+      verificationIcon = 'integr8ly-alert-icon far fa-times-circle';
+    }
     return (
-      <Alert type="info" className="integr8ly-module-column--steps_alert-blue" key={blockId}>
+      <div className={verificationClasses} key={`verification-${blockId}`}>
```

Comment:
Color doesn't change regardless of the selection (see e.g. here: https://tutorial-web-app-webapp.apps.pbraun-9f90.openshiftworkshop.com/tutorial/walkthrough-2/task/0)

Reply:
@pb82 I only added this commit recently, do you have it locally? It seems to be working for me

Reply:
@aidenkeating Nvm. it's working. The version i deployed didn't have that commit.